### PR TITLE
LITTIL-126 - Add contact person entity

### PR DESCRIPTION
**Pull request by roelandkulk**

### CODE REVIEW

File: src/main/resources/ValidationMessages.properties [hunk 1]
```diff
@@ -9,8 +9,9 @@ GuestTeacher.locale.required=The locale of the guest teacher is required.
 School.name.required=The name of the school is required.
 School.address.required=The address of the school is required.
 School.postalCode.required=The postal code of the school is required.
-School.contactPersonName.required=The name of the contact person is required.
 School.contactPersonEmail.invalid=The email address of the contact person is invalid.
 School.contactPersonEmail.required=The email address of the contact person is required.
```
Comment (roelandkulk) [inline src/main/resources/ValidationMessages.properties:13]: Should the School.contactPersonEmail, GuestTeacher.email & GuestTeacher.locale be removed? I believe they are not used anymore.

**Comment by roelandkulk** [comment]
When deleting a School it currently leaves an orphaned ContactPerson and also a Location. Should these be deleted when a School is deleted?

File: src/main/resources/ValidationMessages.properties (hunk 1 above)
Comment (pschildkamp) [reply src/main/resources/ValidationMessages.properties:13]: You can check if this is the case. But I think you are right. You can remove those.

**Comment by pschildkamp** [comment]
> When deleting a School it currently leaves an orphaned ContactPerson and also a Location. Should these be deleted when a School is deleted?

Yes, you should cascade the delete operation.

File: src/main/resources/ValidationMessages.properties [hunk 2]
```diff
@@ -9,8 +9,9 @@ GuestTeacher.locale.required=The locale of the guest teacher is required.
...
+School.firstName.required=The first name of the contact person is required.
```
Comment (pschildkamp) [inline src/main/resources/ValidationMessages.properties:14]: Arent you forgetting the School.prefix.required for the post DTO?

**Review by pschildkamp — CHANGES_REQUESTED** [review]
As you mentioned yourself. Please add the functionality that the contact person will be deleted when the parent school entity is deleted.

File: src/main/resources/ValidationMessages.properties (hunk 2 above)
Comment (roelandkulk) [reply src/main/resources/ValidationMessages.properties:14]: I don't think the prefix should be required.

Comment (pschildkamp) [reply src/main/resources/ValidationMessages.properties:13]: You are 100 percent right.

**Review by pschildkamp — APPROVED** [review]
And again, great stuff!!

**Pull request merged**